src/applications/caregivers/tests/unit/components/FormPages/FacilityConfirmation.unit.spec.jsx

```diff
@@ -15,30 +17,27 @@ describe('CG <FacilityConfirmation>', () => {
   const goForward = sinon.spy();
   const goToPath = sinon.spy();
 
-  const defaultProps = {
-    data: {
-      'view:plannedClinic': {
-        veteranSelected: selectedFacility,
-        caregiverSupport: caregiverFacility,
+  const subject = ({
+    selected = selectedFacility,
+    assigned = caregiverFacility,
+  } = {}) => {
+    const props = {
+      data: {
+        'view:plannedClinic': {
+          veteranSelected: selected !== null ? selected : null,
+          caregiverSupport: assigned !== null ? assigned : null,
+        },
       },
-    },
-    goBack,
-    goForward,
-    goToPath,
-  };
-
-  const subject = (props = defaultProps) => {
-    const { getByText, getByRole } = render(
-      <FacilityConfirmation {...props} />,
-    );
-
+      goBack,
+      goForward,
+      goToPath,
+    };
+    const { queryByText } = render(<FacilityConfirmation {...props} />);
     const selectors = () => ({
-      formNavButtons: {
-        back: getByText('Back'),
-        forward: getByText('Continue'),
-      },
+      backBtn: queryByText('Back'),
+      continueBtn: queryByText('Continue'),
     });
-    return { selectors, getByRole, getByText };
+    return { selectors, queryByText };
```

Comment:
`getByText` actually fails in the test if the value is not found. Since we want to test to ensure if the values exist or not, we want to `queryByText` instead of running the `getByText` method.